docs/jenkins.md

```diff
@@ -0,0 +1,214 @@
+# Jenkins
+
+The Foreman project maintains its own Jenkins instance for continuous integration at https://ci.theforeman.org/
+
+It runs the following types of tests:
+
+* Unit tests on the main develop and all supported stable branches of core Foreman projects
+* Pull request tests against core Foreman projects
+* Unit and pull request tests for many Foreman plugins
+* Nightly and release package building and release processes
+* Package smoke and integration testing
+* Package jobs for Pulpcore
+
+## Quick reference for maintainers
+
+Current PR test jobs (used on Foreman itself) support these commands:
+
+* `ok to test` - run tests for an unknown user, if the code within the patch is not malicious
+* `add to PR whitelist` - automatically run all PRs in future from this user
+* `[test STATUS-NAME]`, e.g. `[test foreman]` to re-run a particular set of tests
+
+Older PR tests, those that use the status name "default", supports this command:
+
+* `[test]` - run tests for an unknown user, or re-run all tests
```

Comment:
I think we got rid of those and now only use GHPRB. The old PR Processor code has been removed so I can't imagine this is still relevant.